ecommerce_extensions/templates/payment/pending_status.html

```diff
@@ -0,0 +1,112 @@
+    <title>{% block title %}Status de Pago{% endblock title %}</title>
+
+    {% compress css %}
+        {% if main_css %}
+            <link rel="stylesheet" href="{{ main_css }}" type="text/x-scss">
+        {% else %}
+            <link rel="stylesheet" href="{% static 'css/base/main.css' %}" type="text/x-scss">
+        {% endif %}
+    {% endcompress %}
+
+    {% compress css %}
+        {# This block is separated to better support browser caching. #}
+        {% block stylesheets %}
+        {% endblock %}
+    {% endcompress %}
+</head>
+<body>
+{% block skip_link %}
+{% endblock skip_link %}
+
+{% block navbar %}
+    {% include 'edx/partials/_staff_navbar.html' %}
+{% endblock navbar %}
+
+{% block info_message %}
+    {% include 'edx/partials/_info_message.html' %}
+{% endblock info_message %}
+
+{% block content %}
+{% endblock content %}
+
+{% block footer %}
+  {% include 'edx/footer.html' %}
+{% endblock footer %}
+
+{# Translation support for JavaScript strings. #}
+<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
+
+<script type="text/javascript">
+    var initModelData = {{ analytics_data|safe }};
+</script>
+
+{% compress js %}
+    <script src="{% static 'bower_components/requirejs/require.js' %}"></script>
+    <script src="{% static 'js/config.js' %}"></script>
+
+     Note: django-compressor does not recognize the data-main attribute. Load the main script separately.
+    <script src="{% static 'js/common.js' %}"></script>
+{% endcompress %}
+
+{% if optimizely_snippet_src %}
+  <script src="{{ optimizely_snippet_src }}"></script>
+{% endif %}
+
+{% comment %}
+    TODO: Remove once this has a MFE, and asking the lms if there is a discount happens in it
+    This was added in order to have the basket page run javascript with context variables. Such javascript
+    cannot be compressed as the variables will change.
+{% endcomment %}
+{% block pre_app_js %}
+{% endblock %}
+
+{% if msg == "pending" %}
+    <div class="receipt container content-container">
+        <h2 class="thank-you">{% trans "Esperando la confirmación de pago." as tmsg %}{{ tmsg | force_escape }}</h2>
+        <div class="confirm-message">
+            Gracias, su orden estara lista cuando el pago sea confirmado.
+        </div>
+    </div>
+{% elif msg == "closed" %}
+    <div class="receipt container content-container">
+        <h2 class="thank-you">{% trans "Su pago ha sido rechazado." as tmsg %}{{ tmsg | force_escape }}</h2>
+        <div class="confirm-message">
+            Hubo un problema a la hora de validar su pago y no pudo concretarse.
+        </div>
+    </div>
+{% elif msg == "error" %}
+    <div class="receipt container content-container">
+        <h2 class="thank-you">{% trans "No se encontro ningun pago registrado." as tmsg %}{{ tmsg | force_escape }}</h2>
+        <div class="confirm-message">
+            La orden que busca no existe.
```

Comment:
English is the default language, if there is not order this should return 404

Reply:
solved